java/ehcache/Portfile

```diff
@@ -1,53 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem 1.0
+PortGroup               java 1.0
+PortGroup               github 1.0
+
+github.setup            ehcache ehcache3 3.8.1 v
+github.tarball_from releases
+name                    ehcache
+distname                ${name}-${version}
+
+categories              java
+maintainers             nomaintainer
+platforms               darwin
+license                 Apache-2
+
+description             Ehcache is a pure Java, in-process object cache.
+long_description        Ehcache is a pure Java, in-process cache with the \
+                        following features: \
```

Comment:
Almost there. This line should be moved left to column 4 also.

Reply:
I've done the same, Sir. May please review it. Thanks.